app/Repositories/Repo.php

```diff
@@ -19,8 +19,9 @@ class Repo
      * @param array $multiValueFilter
      * @return Builder
      */
-    public static function getRecords(string $table, array $columns=[] , array $filters=[] , array $multiValueFilter=[]):Builder{
-        if(empty($columns) || in_array('*' , $columns))
+    public static function getRecords(string $table, array $columns = [], array $filters = [], array $multiValueFilter = []): Builder
```

Comment:
as $filters and $multiValueFilter variables have no role in getRecords body, you can remove them

Reply:
they are passed to filter and filterMultipleValue static functions to use where and whereIn in queries